### PR TITLE
[IMP] email_marketing: getting started

### PR DESCRIPTION
**Pull request by samueljlieber**
This PR tests the push process of the US doc team. 

## The following process created this PR:

Updated base branch
```
git checkout 16.0
git pull
git log --oneline
```
Created feature branch
```
git checkout -b 16.0-us-docteam-push-sali
```
Stage and commit edits
```
git add .
git commit -m "[IMP] email_marketing: getting started"
```
Push to remote
```
git push origin +16.0-us-docteam-push-sali
```

### CODE REVIEW

**Comment by samueljlieber** [comment]
Hi @saliodootest this PR is ready for your review!

File: content/applications/marketing/email_marketing.rst [hunk 1]
```diff
@@ -5,6 +5,8 @@
 Email marketing
 ===============
 
+This is the first change to the document. It a rReally good sentence!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
```
Comment (saliodootest) [inline content/applications/marketing/email_marketing.rst]: This sentence is really messed up dude.
```suggestion
This is the first change to the document. It is a really good sentence!
```

**Review by saliodootest — CHANGES_REQUESTED** [review]
Hi @samueljlieber please find my suggestions below...

**Comment by samueljlieber** [comment]
I cannot `git pull --rebase` because there is no tracking information for the current branch.

I can resolve this by first running `git branch --set-upstream-to=origin/16.0-us-docteam-push-sali`

**Comment by samueljlieber** [comment]
Squashed @saliodootest's review commit and added a sentence in https://github.com/samueljlieber/dev-documentation/commit/0520dba8c8bea9250ad6b0ef1e1c05415b7e502d using `git push origin +16.0-us-docteam-push-sali`.

**Review by saliodootest — APPROVED** [review]
Looks good to me

File: content/applications/marketing/email_marketing.rst [hunk 2]
```diff
@@ -5,6 +5,9 @@
 Email marketing
 ===============
 
+This is the first change to the document. It is a really good sentence!
+I added this sentence too because I thought it was cool.
```
Comment (saliodootest) [inline content/applications/marketing/email_marketing.rst:9]: BAD SENTENCE I remove
```suggestion
```

**Review by saliodootest — COMMENTED** [review]
I like cookies not bad sentences.

**Comment by samueljlieber** [comment]
I am going to make a commit locally ignoring your commit here, and try to push up 😈

**Comment by samueljlieber** [comment]
I did not have any issues overwriting @saliodootest 's commit with my changes with `git push origin +16.0-us-docteam-push-sali`

**Pull request closed without merging**